packages/manager/modules/billing/src/payment/method/add/translations/Messages_fr_FR.json

```diff
@@ -9,7 +9,6 @@
   "billing_payment_method_add_selected_original_credit_card_redirect": "En cliquant sur le bouton \"Ajouter\", un document sera téléchargé. Veuillez suivre la procédure décrite dans le document pour valider le moyen de paiement.",
   "billing_payment_method_add_credit_card_selected": "Votre carte bancaire sera activée immédiatement.",
   "billing_payment_method_add_credit_card_selected_explain_generic": "Pour enregistrer une nouvelle carte bancaire, une empreinte financière est faite auprès de votre organisme bancaire afin de valider la saisie et la validité de votre carte. Aucun montant ne sera prélevé.",
-  "billing_payment_method_add_bank_account_selected": "Votre compte bancaire sera validé entre 24 et 48h à partir du moment où vous aurez signé la procédure (document qui sera téléchargeable à la fin de l'ajout du compte bancaire).",
```

Comment:
This label can be removed from all other languages as well

Reply:
@antleblanc had mentioned that the automated task would take care of removing the corresponding labels in other language files. So it is better to remove only from fr files.

Reply:
Yes, I confirm.
Changes are only allowed on `**/Messages fr_FR.json` files.

For all other languages/translations, there's an automated process.